src/QY2Styler.cc

```diff
@@ -301,7 +301,7 @@ void QY2Styler::registerChildWidget( QWidget * parent, QWidget * widget )
 {
     // Don't use yuiDebug() here - deadlock (reason unknown so far) in thread handling!
 
-    qDebug() << "Registering " << widget << " for parent " << parent << endl;
+    qDebug() << "Registering " << widget << " for parent " << parent << Qt::endl;
```

Comment:
it is not clear for me why here change is required and e.g. at https://github.com/libyui/libyui-qt/pull/127/files#diff-f5ec953a1b429e30261a13ae62c10577R245 it uses plain endl.

Reply:
I understand QTextStream used to have its own ::endl() which is now gone.